File: sdk/python/examples/authenticated.py
from aegis import AegisClient

client = AegisClient(base_url="http://localhost:8080", api_key="phase14-token")
print(f"authenticated={str(client.posture.authenticated).lower()}")
result = client.run(language="bash", code="echo authenticated sdk")
print(result.stdout, end="")

File: sdk/typescript/examples/authenticated.ts
import { AegisClient } from "@aegis/sdk";

const client = new AegisClient({ baseUrl: "http://localhost:8080", apiKey: "phase14-token" });
console.log(`authenticated=${String(client.posture.authenticated)}`);
const result = await client.run({ language: "bash", code: "echo authenticated typescript sdk" });
process.stdout.write(result.stdout);
